bundles/org.openhab.binding.openthermgateway/src/main/java/org/openhab/binding/openthermgateway/internal/MessageType.java

```diff
@@ -0,0 +1,12 @@
+package org.openhab.binding.openthermgateway.internal;
+
+public enum MessageType {
+    ReadData, // 000
```

Comment:
see above, `READ_DATA`

Reply:
Some of the formatting follows the OpenTherm specification standards, perhaps I could do that with some comment to make sure the mapping to the opentherm messages is clear.